oa/components/README.md

```diff
@@ -0,0 +1,196 @@
+
+**Enable/Disable GTI service**
+
+It's possible to disable any of the reputation services mentioned above, all it takes is to remove the configuration for the undesired service in gti_config.json. To learn more about this see the next section.
+
+**Configuration**
+
+- gti_config.json: Stores a list of reputation services to call during oni-oa execution. Also it contains a list of columns to check
+ reputation. gti_config.json looks like this:
+
+        {
+            "gti":{
+                "server" : "<server>",
+                "user" : "<user>",
+                "password" : "<password>",
+                "ci" : "{\"ci\":{\"cliid\":\"<cliid>\", \"prn\":\"<prn>\", \"sdkv\":\"1.0\", \"pv\":\"1.0.0\", \"pev\":1, \"rid\":1, \"affid\":\"0\"},\"q\":[###QUERY###]}",
+                "refclient" : "<refclient location>",
+                "category_file":"<gti_cat_codes file location>"
+            },
+            "fb":{
+                "app_id" : "<app_id>",
+                "app_secret" : "<app_secret>"
+            }
+        }
+    Where
+    - gti: McAfee GTI service connection details.
+        - server: string with McAfee GTI server.
+        - user: string with McAfee GTI user.
+        - password: string with McAfee GTI password.
+        - ci: request body for GTI service. This string parameter is formatted as json content,
+        be careful with removing scape characters for single quotes and double quotes.
+
+        Do not remove, replace or modify the label ###QUERY###, it's a special placeholder and is required for
+        reputation.py to work.
+        - prn: you can give a name to your Oni instance like "CompanyX-Oni" or you can just leave as Oni.
+        - refclient: absolute path for restclient. It should be the path where the *restclient* file is
+        located without ending backslash i.e.
+
+                /home/solution-user/refclient/restclient
+
+        - category_file: absolute path for the gti category and functional names catalog. This file is provided by McAfee upon hiring this service.
+        This file should be a comma separated text file, including headers for the following columns:
+            - Category code: int
+            - Category name: string
+            - Functional group: string
+
+    - fb: facebook [ThreatExchange](https://developers.facebook.com/products/threat-exchange) service connection details.
+        - app_id: App id to connect to ThreatExchange service.
+        - app_secret: App secret to connect to ThreatExchange service.
+
+###IANA
+Internet Assigned Numbers Authority codes translation module.
+
+**Configuration**
+
+- iana_config.json:  Stores the location of codes translation files. By default, oni-oa iana module comes with 4 csv files
+with different code types for DNS and HTTP queries and usually they will be located in ~/oa/components/iana. These files can be

```

Comment:
In ~/oa/components/iana, is "~" referring to the oni-ao installation folder? It might be confused with the user home directory.